tests/integration_tests/build/test_build.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Tests to check several aspects of the build"""
+
+import re
+import subprocess
+
+
+def test_release_debuginfo(microvm_factory):
+    """Ensure the debuginfo file has the right ELF sections"""
+    fc_binary = microvm_factory.fc_binary_path
+    debuginfo = fc_binary.with_suffix(".debug")
+    stdout = subprocess.check_output(
+        ["readelf", "-S", str(debuginfo)],
+        encoding="ascii",
+    )
+    matches = {
+        match[0]
+        for match in re.findall(r"\[..] (\.(\w|\.)+)", stdout, re.MULTILINE)
+    }
+    needed_sections = {
+        ".debug_aranges",
+        ".debug_info",
+        ".debug_abbrev",
+        ".debug_line",
+        ".debug_frame",
+        ".debug_str",
+        ".debug_ranges",
+    }
+    missing_sections = needed_sections - matches
+    assert missing_sections == set()
```

Comment:
if you do another revision for any reason, maybe nit:

```suggestion
    assert not missing_sections, missing_sections
```

so that pytest prints out the full set (because the condition part gets trimmed when printing failures). But not very important given that for this test we don't really care _what's_ inside the `missing_sections`.

Reply:
This is what I get now.

```
>>>>>>>>>>>>>>>>>>>>>>>>>>>> traceback >>>>>>>>>>>>>>>>>>>>>>>>>>>>>
integration_tests/build/test_build.py:33: in test_release_debuginfo
    assert missing_sections == set()
E   AssertionError: assert {'.debug_foo'} == set()
E     Extra items in the left set:
E     '.debug_foo'
E     Full diff:
E     - set()
E     + {'.debug_foo'}
```

Do you mean if the diff is too large, we don't see everything? Maybe we should enable `-v` in pytest options.

Reply:
Mh, maybe pytest actually does it more cleverly for sets, but I remember that for dictionaries that was indeed the case (diff too large, and pytest at some point just goes `...`)

Reply:
Yeah I think with `-v` or `-vv` it will not omit info.